web-console/src/lib/compositions/useHashPart.ts

```diff
@@ -1,12 +1,20 @@
 import { tuple } from '$lib/functions/common/tuple'
-import { useRouter } from 'next/router'
+import { useParams } from 'next/navigation'
 import { useEffect, useState } from 'react'
 
 // https://stackoverflow.com/questions/69343932/how-to-detect-change-in-the-url-hash-in-next-js
 export const useHashPart = () => {
-  const router = useRouter()
   const [hash, setHash] = useState(('window' in globalThis && window.location.hash) || '')
 
+  // https://github.com/vercel/next.js/discussions/49465#discussioncomment-5845312
+  const params = useParams()
+  useEffect(() => {
+    const newHash = window.location.hash.split('#')[1] ?? ''
```

Comment:
seems like this will cause an error if we index with `[1]` but the list does not contain more than 1 element?

Reply:
That's expected and is caught with `?? ''`